test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,9 +1,55 @@
+---
 # NXAPI is enabled differently on NX-OS platforms.
 # Try both commands to enable NXAPI and ignore any errors.
 - name: enable nxapi on remote device
   nxos_config:
     lines:
-      - feature cli
-      - feature cli sandbox
+      - feature nxapi
+      - feature nxapi sandbox
```

Comment:
I can't find a reference to `feature nxapi sandbox` being a valid command on any platform. Did you mean `nxapi sandbox` or is there something I'm missing here?